config/loss/binary/dice.yaml

```diff
@@ -0,0 +1,5 @@
+# @package _global_
+loss:
+    _target_: segmentation_models_pytorch.losses.DiceLoss
+    mode: binary
+    ignore_index: ${dataset.ignore_index}
```

Comment:
I don't think these missing newlines will cause trouble. I can add them if anybody's nervous :)

Reply:
where is it passed if removed ?

Reply:
The only script that is removed is losses/__init__.py. It is replaced by a colleciton of yaml, one for each loss. Each yaml points to the class that needs to be instantiated by Hydra. It's the "hydranic" way of things that will also appear once we tackle issue #246